perception/autoware_ground_segmentation/README.md

```diff
@@ -33,15 +33,14 @@ Detail description of each ground segmentation algorithm is in the following lin
 
 ### Node Parameters
 
-| Name                 | Type   | Default Value | Description                           |
-| -------------------- | ------ | ------------- | ------------------------------------- |
-| `input_frame`        | string | " "           | input frame id                        |
```

Comment:
Not really for this PR, but
" " does not evaluate as empty, so I think it goes against its apparent objective
https://github.com/autowarefoundation/autoware.universe/blob/b3caa5cea50a6747ff77f96786396b3176840a74/sensing/autoware_pointcloud_preprocessor/src/filter.cpp#L274

@badai-nguyen  what do you think?

Reply:
@knzo25 I'm so sorry, I missed your mention here let have a look.